src/show/pages/public/Standings/index.jsx

```diff
@@ -1,70 +1,70 @@
-              place='1'
-              points='176'
-            />
-            <ConstructorStandingCard
-              logo='https://logodownload.org/wp-content/uploads/2014/04/mercedes-benz-logo-1-1.png'
-              place='2'
-              points='176'
-            />
-            <ConstructorStandingCard
-              logo='https://logodownload.org/wp-content/uploads/2014/04/mercedes-benz-logo-1-1.png'
-              place='3'
-              points='176'
-            />
-            <ConstructorStandingCard
-              logo='https://logodownload.org/wp-content/uploads/2014/04/mercedes-benz-logo-1-1.png'
-              place='4'
-              points='176'
-            />
-            <ConstructorStandingCard
-              logo='https://logodownload.org/wp-content/uploads/2014/04/mercedes-benz-logo-1-1.png'
-              place='5'
-              points='176'
-            />
-            <ConstructorStandingCard
-              logo='https://logodownload.org/wp-content/uploads/2014/04/mercedes-benz-logo-1-1.png'
-              place='6'
-              points='176'
-            />
-            <ConstructorStandingCard
-              logo='https://logodownload.org/wp-content/uploads/2014/04/mercedes-benz-logo-1-1.png'
-              place='7'
-              points='176'
-            />
-            <ConstructorStandingCard
-              logo='https://logodownload.org/wp-content/uploads/2014/04/mercedes-benz-logo-1-1.png'
-              place='8'
-              points='176'
-            />
-            <ConstructorStandingCard
-              logo='https://logodownload.org/wp-content/uploads/2014/04/mercedes-benz-logo-1-1.png'
-              place='9'
-              points='176'
-            />
-            <ConstructorStandingCard
-              logo='https://logodownload.org/wp-content/uploads/2014/04/mercedes-benz-logo-1-1.png'
-              place='10'
-              points='176'
-            />
+    <div className='flex w-full flex-col'>
+      <div className='flex w-full  overflow-y-auto flex-col bg-primary'>
+        <div className='flex w-full  bg-white h-52 overflow-x-auto'>
+          <div className='flex ml-5 mt-3 flex-col text-charade text-2xl font-bold antialiased'>
+            Constructors Standing
+            <div className='flex flex-row'>
+              <ConstructorStandingCard constructor={constructor} />
+              <ConstructorStandingCard constructor={constructor} />
+              <ConstructorStandingCard constructor={constructor} />
+              <ConstructorStandingCard constructor={constructor} />
+              <ConstructorStandingCard constructor={constructor} />
+              <ConstructorStandingCard constructor={constructor} />
+              <ConstructorStandingCard constructor={constructor} />
+              <ConstructorStandingCard constructor={constructor} />
+              <ConstructorStandingCard constructor={constructor} />
+              <ConstructorStandingCard constructor={constructor} />
```

Comment:
You can have an array of constructors, then map through each of them to display a <ConstructorStadingCard />